src/screens/Home/index.tsx

```diff
@@ -30,7 +28,6 @@ export default function Home() {
   const styles = getStyles();
   const { client } = useAuth();
   const theme = useTheme() as MyMD3Theme;
-
   const [activeTab, setActiveTab] = useState(1);
```

Comment:
it would be better to use `enum`

Reply:
Yes. agree.  
Currently `setActiveTab` is getting `index<number> ` from `<CustomTab onTabChange{setActiveTab} />`.
And CustomTab component is being used in 4 different screens. 

If we change this, we have to change another  4 screens as well. 

can we do this later again? :') .